pages/topics/gateway-events.mdx

```diff
@@ -634,6 +635,8 @@ These events directly correspond with a specific action or state change that has
 | [Stage Instance Update](#stage-instance-update)                                   | Stage instance was updated                                                                      |
 | [Stage Instance Delete](#stage-instance-delete)                                   | Stage instance was deleted or closed                                                            |
 | [Typing Start](#typing-start)                                                     | User started typing in a channel                                                                |
+| [User Application Remove](#user-application-remove)                               | User uninstalled application                                                                    |
+| [User Connections Update](#user-connections-update)                               | User connection was created, updated, or deleted                                                |
```

Comment:
Please move these two alphabetically below user update (and the structs below)

Reply:
yeah the priority is categorizing and then alphabetical, which is why e.g. thread events are under channel events